ts/screens/profile/CalendarsPreferencesScreen.tsx

```diff
@@ -40,6 +43,11 @@ class CalendarsPreferencesScreen extends React.PureComponent<Props, State> {
     this.props.navigation.goBack();
   };
 
+  private onCalendarRemove = () => {
+    this.props.preferredCalendarRemoveSuccess();
+    this.props.navigation.goBack();
```

Comment:
imho automatic go back could be annoying, i.e: I have 3 calendars and  I want to select another calendar different from the current one, so I could deselect the current one and then select the new one.
In this way I will be pushed out when I deselect the current calendar.

What do you think?

Reply:
Yes, I completely agree!
I just left the logic how it was previously implemented.
I'm going to remove the go back from remove and save calendar

Reply:
@Undermaken fixed 4244d4a